Cargo.toml

```diff
@@ -10,6 +10,7 @@ members=[
     "lrpar/examples/calc_actions",
     "lrpar/examples/calc_ast",
     "lrpar/examples/calc_parsetree",
+    "lrpar/examples/calc_storaget",
```

Comment:
Whoops, this one doesn't actually exist in the repository, and adding it belongs to a separate PR most likely.